pom.xml

```diff
@@ -37,34 +37,28 @@
   </pluginRepositories>
 
   <properties>
+    <revision>2.32.0</revision>
+    <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.289.3</jenkins.version>
-    <java.level>8</java.level>
-    <artifactory-maven-plugin.version>2.6.1</artifactory-maven-plugin.version>
-    <project.build.testSourceDirectory>src/test/java</project.build.testSourceDirectory>
+    <jenkins.version>2.387.3</jenkins.version>
```

Comment:
https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/

Reply:
Indeed, I was worried that you took a too recent baseline. It is actually relatively reasonable and still covers most of the user base of the plugin as per https://stats.jenkins.io/pluginversions/workflow-cps-global-lib-http.html